src/current/_includes/common/tls-bad-cipher-warning.md

```diff
@@ -1,4 +1,19 @@
+{% comment %}This include is used only in v22.2+ docs and in one CC page (which should use the 23.1+ variant){% endcomment %}
 {{site.data.alerts.callout_info}}
+{% if page.version.version == "v22.2" %}
+CockroachDB clusters support TLS 1.2 or TLS 1.3 encryption for SQL clients. However, the following less-secure TLS 1.2 cipher suites are not available in v22.2.0 - v22.2.3, and are are rejected by default in v22.2.4 and above, in accordance with the IETF's recommended cipher list defined in [RFC 8447](https://datatracker.ietf.org/doc/html/rfc8447):
+
+- `TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA`
+- `TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA`
+- `TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA`
+- `TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA`
+- `TLS_RSA_WITH_AES_128_GCM_SHA256`
+- `TLS_RSA_WITH_AES_256_GCM_SHA384`
+- `TLS_RSA_WITH_AES_128_CBC_SHA`
+- `TLS_RSA_WITH_AES_256_CBC_SHA`
+
+SQL clients, intermediate proxies, or load balancers that do not support any cipher suites that CockroachDB supports will be unable to connect to CockroachDB clusters. To allow SQL connections using the deprecated cipher suites, set the `COCKROACH_TLS_ENABLE_OLD_CIPHER_SUITES` environment variable to `true` for your `cockroach start` command. This mode is not recommended, unless you must use a client, intermediate proxy, or load balancer that doesn't support any of the more secure cipher suites. This environment variable is not available in v22.2.0 - v22.2.3.
+{% else %}
```

Comment:
Oh, this text was already here and you're just adding some very precise notes about patch versions? I see. 

I'll leave my general comments but maybe they don't need to happen right now. I still think that putting patch versions in the first paragraph makes it confusing and all I'd do is add the last sentence "This environment variable is not available..." to the last paragraph.

Reply:
It ended up being feasible to externalize the list.